thoth/slo_reporter/templates/pypi_knowledge_graph.html

```diff
@@ -0,0 +1,21 @@
+{% include "base.html" %}
+{% block title %} <strong> PyPI Knowledge Graph </strong> {% endblock %}
+{% block classes %}
+<thead>
+    <tr>
+        <th>Parameter</th>
```

Comment:
do we need this? it is confusing the user/reader

Reply:
Didn't we want to show differences in terms of global knowledge?

Reply:
It's not activated yet in any case, as metrics are not yet scraped for that.